executor/server.go

```diff
@@ -0,0 +1,88 @@
+package executor
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/ngaut/log"
+	"github.com/sergi/go-diff/diffmatchpatch"
+)
+
+// DoDiff executes sqls in the driver configured in exec. If there is an inconsistent sql execution result,
+// it will directly return an error, and the error message contains inconsistencies.
+// If the execution results are the same, no error is returned.
+// handleResultsFn is used to do special processing on the results after executing each statement.
+func DoDiff(exec *Executor, sqls []string, handleResultsFn func(strs []string)) error{
```

Comment:
It's better to make the `DoDiff` as a method of `Executor`? (not a strong opinion).

Reply:
Please format this file.